downstream/modules/platform/con-edge-manager-enroll-meth.adoc

```diff
@@ -0,0 +1,15 @@
+[id="edge-manager-enroll-meth"]
+
+= Enrollment methods
+
+You can provision the enrollment endpoint and certificate to the device in the following ways:
+
+Early binding:: You can build an operating system image that includes the enrollment endpoint and certificate. 
+Devices using an early binding image can automatically connect to the defined  service to request enrollment, without depending on any provisioning infrastructure. 
```

Comment:
It looks like there's an extra space between ...defined  service... in this sentence.